src/FirmwarePlugin/APM/ArduCopterFirmwarePlugin.cc

```diff
@@ -136,3 +136,44 @@ bool ArduCopterFirmwarePlugin::vehicleYawsToNextWaypointInMission(const Vehicle*
     return true;
 }
 
+void ArduCopterFirmwarePlugin::guidedModeGotoLocation(Vehicle* vehicle, const QGeoCoordinate& gotoCoord)
+{
+    if (qIsNaN(vehicle->altitudeRelative()->rawValue().toDouble())) {
+        qgcApp()->showMessage(QStringLiteral("Unable to go to location, vehicle position not known."));
+        return;
+    }
+
+    setGuidedMode(vehicle, true);
+
+    if (vehicle->capabilityBits() & MAV_PROTOCOL_CAPABILITY_COMMAND_INT) {
```

Comment:
This only applies to sending COMMAND_INT (https://mavlink.io/en/messages/common.html#COMMAND_INT) mavlink messages. It doesn't really apply to other messages are far as I know. At least not accordint to the mavlink spec.

Reply:
@hamishwillee Not sure if this is another point of confusion/discussion in the spec or not?